src/index.js

```diff
@@ -0,0 +1,31 @@
+import Personnummer from './personnummer';
+
+/**
+ * Parse personnummer.
+ *
+ * @param {string} ssn
+ */
+const parse = (ssn, options) => new Personnummer(ssn, options);
```

Comment:
If wanted, this could be a static method on the `Personnummer` class instead, but having it as a separate method like this might actually be a better idea in this case.

Reply:
Hm.. yeah. Not sure what's best. Maybe just move to static methods for both valid and parse.

Reply:
Added static methods and removed functions

```
const Personnummer = require('personnummer')
```